src/tools/sfdk/command.cpp

```diff
@@ -262,10 +281,16 @@ class VirtualMachinePropertiesAccessor : public PropertiesAccessor
         } else if (name == VM_STORAGE_SIZE_MB) {
             if (!parsePositiveInt(&m_storageSizeMb, value, errorString))
                 return Failed;
-            if (m_storageSizeMb < m_vm->storageSizeMb()) {
+            if ((m_storageSizeMb < m_vm->storageSizeMb())
+                && !(m_vm->features() & VirtualMachine::ShrinkStorageSize)) {
```

Comment:
Should be indented once more. I would prefer omitting braces on the left side of `&&`. Similar below.

Reply:
fixed